sources/models/src/lib.rs

```diff
@@ -242,6 +242,14 @@ struct MetricsSettings {
     service_checks: Vec<String>,
 }
 
+// CloudFormation settings
+#[model]
+struct CloudFormationSettings {
+    signal: bool,
```

Comment:
I'm wondering if we should call this `should_send` instead of `signal`.

The reason I'm thinking about that is that it might be useful in the future to have a setting that forces cfsignal to explicitly send `FAILURE` even if all the relevant systemd units were successful.  If, for example, there were a bootstrap container that was expected to execute some form of setup job and that failed, the bootstrap container could then indicate its failure all the way back out to CloudFormation.  In that scenario, it might be useful to have a `signal` setting with the values of `SUCCESS`/`FAILURE` and a `should_send` boolean setting that controls whether cfsignal sends anything.